CHANGELOG.md

```diff
@@ -623,4 +627,4 @@ that would be used if a command were executed.
 ## `3.0.0`
 
 - Rename package from "@brightside/imperative" to "@zowe/imperative".
-- Change name of config option "credential-manager" to "CredentialManager".
\ No newline at end of file
+- Change name of config option "credential-manager" to "CredentialManager".
```

Comment:
Interesting how this has always been indented 😋

Reply:
Hi Fernando, that's an extra new line in the end my IDE added if that's what you asking :)

Reply:
Oh no, not at all, I meant more the contents themselves are 2-space indented 😋 
But I{m afraid that{s just on the GH PR view since it adds the `- | +` signs.
![image](https://user-images.githubusercontent.com/37381190/173063400-4ecb34fb-ec77-4f0b-8925-13efb35168f9.png)

Sorry for the confusion 😅